ftdc/cmd/parser.go

```diff
@@ -27,6 +32,26 @@ type gnuplotWriter struct {
 	metricFiles map[string]*os.File
 
 	tempdir string
+
+	options graphOptions
+}
+
+type graphOptions struct {
+	minTimeSeconds int64
+	maxTimeSeconds int64
+}
+
+func defaultGraphOptions() graphOptions {
+	return graphOptions{
+		minTimeSeconds: 0,
+		maxTimeSeconds: math.MaxInt64,
+	}
+}
+
+func nolintPrintln(str ...any) {
+	// This is a CLI. It's acceptable to output to stdout.
```

Comment:
> This is a CLI

Obviously there are tools like https://github.com/urfave/cli to make writing Go CLIs a lot easier/more ergonomic. I understand the desire to just get something out the door here, but if this CLI program grows at all in complexity, I'd certainly be in favor of making it a bit more like the `viam` CLI and build it with urfave's library. The command-line argument processing and STDOUT/STDERR printing will be much easier to maintain/read through. Again, just a point for posterity, no need to change anything here.

Reply:
> I understand the desire to just get something out the door here, but if this CLI program grows at all in complexity, I'd certainly be in favor of making it a bit more like the viam CLI and build it with urfave's library

If this gets used, I expect to merge* this with the viam-cli. Unless there's product push back.

I might also fight you about urfave's utility. The state sharing across commands becomes more magical. Maybe

Reply:
> If this gets used, I expect to merge* this with the viam-cli. Unless there's product push back.

Sounds good to me. To some degree that merge will probably force you to use urfave, but you're right that urfave does not really support REPL stuff in any meaningful way (will just have to keep the reading from STDIN, although it will likely be pulling from some `io.Reader` on the urfave app object instead of `io.Stdin`).